ehrapy/api/data/_datasets.py

```diff
@@ -109,3 +109,36 @@ def heart_failure(columns_obs_only: dict[str, list[str]] | list[str] | None = No
     )
 
     return adata
+
+def diabetes_130(
+    encoded: bool = False,
+    columns_obs_only: dict[str, list[str]] | list[str] | None = None,
+) -> AnnData:  # pragma: no cover
+    """Loads the diabetes-130 dataset
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Diabetes+130-US+hospitals+for+years+1999-2008
+    Preprocessing: https://github.com/theislab/ehrapy-datasets/tree/main/diabetes_130/diabetes_130.ipynb
+
+    Args:
+        encoded: Whether to return an already encoded object
```

Comment:
The columns_obs_only parameter is missing :) I had forgotten it as well, but added it.